site/server/app/com/fortysevendeg/exercises/utils/OAuth2.scala

```diff
@@ -72,23 +82,24 @@ class OAuth2Controller(ws: WSClient, userService: UserServices) extends Controll
 
           val login = (response.json \ "login").as[String]
           val name = (response.json \ "name").as[String]
-          val github_id = (response.json \ "id").as[Long]
-          val avatar_url = (response.json \ "avatar_url").as[String]
-          val html_url = (response.json \ "html_url").as[String]
+          val githubId = (response.json \ "id").as[Long]
```

Comment:
This is not safe but I understand we are changing this in another PR. All JSON parsing ops should be treated as an applicative builder style where a chunk is reduce to the structure or errors are reported accordingly, similar how the functional builders for JSON in play work.